libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/index.ts

```diff
@@ -167,15 +184,33 @@ class TigerWorkspaceElementsQuery implements IElementsQuery {
                     return client.labelElements.computeLabelElementsPost(elementsRequestWrapped);
                 });
 
-                const { paging, elements } = response.data;
+                const { paging, elements, format } = response.data;
+
+                const refAsValue = objRefToString(this.ref);
+                const granularity = this.getGranularity(refAsValue);
+                const sdkGranularity = toSdkGranularity(granularity);
+                const locale = format?.locale as any;
+                const pattern = format?.pattern as any;
+                const dateValueFormatter = createDateValueFormatter(this.dateFormatter);
 
                 return {
-                    items: elements.map(
-                        (element): IAttributeElement => ({
+                    items: elements.map((element): IAttributeElement => {
+                        const objWithFormattedTitle = sdkGranularity
```

Comment:
Condition `format && sdkGranularity` would be safer here.

Reply:
added